bot/cogs/source.py

```diff
@@ -42,19 +50,26 @@ def get_command_url(command: commands.Command) -> str:
         """Make up the url for the source of the command."""
         # Get the source code
         src_code_object = command.callback.__code__
-
-        # Get module name and replace . with /
-        module_name = command.callback.__module__
-        module_name = module_name.replace(".", "/") + ".py"
+        file_path = src_code_object.co_filename[5:]
```

Comment:
Rapptz's implementation of getting the location was like this: `file_name = src_code_object.co_filename` and `location = os.path.relpath(filename).replace('\\', '/')`. I think, should work; will do some tests in a bit.